src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -98,15 +98,14 @@ public static function register($mode = 0)
             'legacyCount' => 0,
             'otherCount' => 0,
             'remaining vendorCount' => 0,
-            'unsilenced' => array(),
-            'remaining' => array(),
-            'legacy' => array(),
-            'other' => array(),
-            'remaining vendor' => array(),
+            'unsilenced' => [],
+            'remaining' => [],
+            'legacy' => [],
+            'other' => [],
+            'remaining vendor' => [],
```

Comment:
the bridge in 4.2 needs to remain compatible with PHP 5.3, this should be reverted